charts/fairspace/values.yaml

```diff
@@ -12,7 +12,7 @@ external:
     realm:
     clientSecret:
 
-# Setup ingress for fairspace components
+# Setup ingress for Metatree components
```

Comment:
All the names here (values) are still Fairspace, why changing the comments?

Reply:
that's a valid point :)